Security/Data_Classification.mediawiki

```diff
@@ -9,21 +9,28 @@ data classification
     <td style="min-width: 25em;">__TOC__</td>
     <td style="vertical-align: top; padding-left: 1em;">
 <span style="background-color: #14892c; border-radius: .25em; color: #ffffff; display: inline-block; font-weight: bold;
-margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">READY</span>
+margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">READY</span> <br>
+
+<br>
 
 The data classification is intended to allow Mozilla to operate effectively in the open while protecting sensitive information.
 
-These data classification levels use the colors of the [[Security/Standard Levels|Standard Levels]].
+These data classification levels use the colors of the [[Security/Standard Levels|Standard Levels]]. <br>
+
+<br>
+
+This classification scheme is solely meant to communicate who the intended audiences of a particular file is, not what type of content is contained within the document.<br>
+
+<br>
 
-If you think there is information that is improperly classified, we encourage you to work with the document owner, using
-this mana page as a reference. If you are unable to resolve this directly, we encourage you to complete the form at
-http://mzl.la/reclassify
+If you think there is information that is improperly classified, we encourage you to work with the document owner, using this page as a reference. If you are unable to come to a resolution, please reach out to the Infrastructure Security team via Slack (@infrasec) or [[mailto:infrastructuresecurity@mozilla.com | email]]. <br>
```

Comment:
I don't think we should (on this public wiki page) ask folks to contact our team. Also our mailing list isn't open to the public.

In the past there was a public Google Drive form that people could fill out that would send an email. It was never used and so I closed it down. That makes me think that there's no need to give users a path to contact us.

Reply:
SGTM

Reply:
See most recent commit in reference to this (https://github.com/mozilla/wikimo_content/pull/143/commits/fa95f9007416494bf25b25c7a0c6b0f157587f17)